prog/dftb+/lib_dftbplus/main.F90

```diff
@@ -1483,18 +1483,20 @@ subroutine reallocateSparseArrays(sparseSize, reks, ham, over,&
 
     integer :: nSpin
 
-    #:call ASSERT_CODE
```

Comment:
The Hellman-Feynman block should probably also use the same construction. Likewise #491

Reply:
`#:call` and `#:block` are the same constructs in Fypp (they call fypp-macro), just using different names for improved readability.

As for `ASSERT_CODE` vs. `DEBUG_CODE`: Former is included, if asserts are active (and should only contain assert related stuff), latter if the build type is debug (and could contain any kind of debug stuff). So in theory they are different. (However, as currently, debug and asserts are turned together on and off, there is no practical difference, I agree.)